ios/browser/api/web_image/web_image.mm

```diff
@@ -51,28 +52,59 @@ - (void)downloadImage:(NSURL*)url
           return;
         }
 
-        if (bitmaps.size() > 1) {
-          NSMutableArray* frame_images = [[NSMutableArray alloc] init];
-          for (const auto& bitmap : bitmaps) {
-            gfx::Image image = gfx::Image::CreateFrom1xBitmap(bitmap);
-            [frame_images addObject:image.IsEmpty() ? [[UIImage alloc] init]
-                                                    : image.ToUIImage()];
-          }
+        completion([WebImageDownloader decodeFrames:bitmaps
+                                              sizes:bitmap_sizes
+                                     animationDelay:2.0],
```

Comment:
This is OK to hard-code as 2?

Reply:
Yeah. I think the iOS side doesn't care about animations atm. I just gave it a default value so it doesn't look bad if it renders animations. Atm, there isn't really a way to determine the speed of a gif through Skia anyway, so any value put here will be random.